backend/src/test/java/com/bakdata/conquery/util/search/QuickSearchTest.java

```diff
@@ -0,0 +1,24 @@
+package com.bakdata.conquery.util.search;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+class QuickSearchTest {
+	@Test
+	public void listItems() {
+		final QuickSearch<Integer> quickSearch = new QuickSearch<>();
+
+		quickSearch.addItem(0,"b");
+		quickSearch.addItem(1,"c");
+		quickSearch.addItem(2,"a");
+
+		assertThat(quickSearch.listItems())
+				.containsExactly(2, 0, 1);
```

Comment:
Okay, ich bin etwas verwirrt. Hätte ich nicht an dieser Stelle gerne die Items in einer nachvollziehbaren Reihenfolge und nicht sortiert nach irgendeinem Keyword?

Reply:
Ja hab nochmal überlegt, hätte gedacht, dass es mehr Sinn macht es in Reihenfolge der keywords zu sortieren aber theoretisch können die sich ja auch widersprechen. Hab einen sinnvollen Comparator in FEValue eingebaut